conformance/e2e-tests/src/client.rs

```diff
@@ -54,7 +54,7 @@ fn tls_handshake_timeout(protocol: &str) -> Result<(), Error> {
 
             assert!(!output.status.success());
             println!("stdout:\n {}", output.stdout);
-            assert!(output.stdout.contains("timed out"));
```

Comment:
this part of the commit diff seems out of place without some justifying comment. It isn't consistent which spelling is used? The next commit also removes both of these contains checks in favour of a 3rd form that's using stderr instead of stdout, but it isn't obvious why this transition state happens before that.